linux/daemon/OSPDaemon_queue.h

```diff
@@ -0,0 +1,20 @@
+#ifndef _OSPDAEMON_QUEUE_H_
+#define _OSPDAEMON_QUEUE_H_	1
+
+#define QUEUE_LEN	25
```

Comment:
Will this length be sufficient in case of batching as well? Or should we revisit this when batching is implemented?